augly/audio/intensity.py

```diff
@@ -201,3 +201,10 @@ def time_stretch_intensity(rate: float = 1.5, **kwargs) -> float:
 
 def to_mono_intensity(metadata: Dict[str, Any], **kwargs) -> float:
     return 0.0 if metadata["src_num_channels"] == 1 else 100.0
+
+# unsure as to how intensity works
+def loop_audio_intensity(n: int = 1, **kwargs) -> float:
```

Comment:
Why not implement this like [the one](https://github.com/facebookresearch/AugLy/blob/main/augly/video/helpers/intensity.py#L160) for `loop` in video?